CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contribution Guide
+
+## Getting Started
+
+### Prerequisites
+Please make sure you have the following programs installed:
+- [Python 3.9](https://www.python.org/downloads/)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads)
+
+### Setting up
+1. Clone your fork.
+2. Create a new branch.
+3. Open any terminal and navigate to the project directory.
+4. Run the `vagrant up` command.
+
+After the dependencies were installed and the VM initialized, you can proceed working on the project code.
+
+## Development Process
+
+### Commits
+- Separate subject from body with a blank line.
+- Limit the subject line to 50 characters.
+- Capitalize the subject line.
+- Do not end subject line with a period.
+- Use the imperative mood in the subject line.
+- Wrap the body at 72 characters.
+- Use the body to explain what and why vs. how.
+
+For more information and examples, please refer to [this guide](https://cbea.ms/git-commit/#seven-rules).
+
+### Issues
+- The title must be clear, and describe what the issue is about.
+- The description of the issue must include a clear explanation about the reasons for doing the work.
+- Status Labels(Optional):
+  - Status/New - An issue that was just created, in addition unlabeled issues are considered new.
+  - Status/Backlog - A issue that was discussed and accepted for work.
+  - Status/In progress - An issue that is being worked on.
+
+### Pull Requests
+- **The Pull Request should not break any of the existing functionality**
+- <ins>Description-</ins> the PR description should explain what changes have been made, and why. Incase it's related to a specific `issue` - it is mandatory to include a link to it.
```

Comment:
You're using underline and dash here here, but on other similar occasions you no underline and colon.